jinahub/encoders/audio/AudioCLIPEncoder/Dockerfile.gpu

```diff
@@ -0,0 +1,18 @@
+FROM jinaai/jina:2-py37-perf
```

Comment:
To support GPU, we should use base image with cuda-driven, e.g., `FROM nvidia/cuda:11.2.1-cudnn8-runtime`

Reply:
Nope, not for pytorch - the pip pytorch package already includes all needed CUDA binaries. Try it yourself.

Reply:
No, when using torch, this base image is working fine. It is installing all cuda stuff by itself. We have done it like that also for the other Torch GPU encoders

Reply:
cool